packages/accordions/src/elements/accordion/Accordion.tsx

```diff
@@ -31,37 +31,39 @@ interface IStaticAccordionExport<T, P>
 }
 
 interface IAccordionProps extends Omit<HTMLAttributes<HTMLDivElement>, 'onChange'> {
-  /** The `aria-level` used to apply heading rank on accordion headers */
+  /** Sets `aria-level` heading rank in the document structure */
   level: number;
-  /** Sets the sections for expansion */
+  /** Sets the expanded sections in a controlled accordion */
   expandedSections?: number[];
-  /** Hides the bottom border under each accordion panel */
+  /** Hides section borders */
```

Comment:
Should we designate "section" as "accordion panel" since that [is the semantic name](https://www.w3.org/TR/wai-aria-practices/#accordion)?

Reply:
I wondered about that. I landed on section after a look at the html and saw that a <section> encapsulated the panel component, and because section was the language used on the website. Happy to change though!

Reply:
Let's use the term "section" as that is consistent with the existing wording used throughout the component page: https://garden.zendesk.com/components/accordion. Also, the same term is used throughout the WAI-ARIA docs.

> Accordion (Sections With Show/Hide Functionality)